distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -17,113 +17,207 @@
-log4j.appender.out.layout.ConversionPattern=%d{ISO8601} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
-log4j.appender.out.file=${karaf.data}/log/ddf.log
-log4j.appender.out.append=true
-log4j.appender.out.maxFileSize=20MB
-log4j.appender.out.maxBackupIndex=10
-
-# Ingest Error File appender
-log4j.appender.ingestError=org.apache.log4j.RollingFileAppender
-log4j.appender.ingestError.layout=org.apache.log4j.PatternLayout
-log4j.appender.ingestError.layout.ConversionPattern=%d{ISO8601} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
-log4j.appender.ingestError.file=${karaf.data}/log/ingest_error.log
-log4j.appender.ingestError.append=true
-log4j.appender.ingestError.maxFileSize=20MB
-log4j.appender.ingestError.maxBackupIndex=10
-
-# Security File appender
-log4j.appender.security=org.apache.log4j.RollingFileAppender
-log4j.appender.security.layout=org.apache.log4j.PatternLayout
-log4j.appender.security.layout.ConversionPattern=[%-5p] %d{ISO8601} | %-16.16t | %-15.20c{1} |  %m%n
-log4j.appender.security.file=${karaf.data}/log/security.log
-log4j.appender.security.append=true
-log4j.appender.security.maxFileSize=20MB
-log4j.appender.security.maxBackupIndex=10
-
-# Solr File appender
-log4j.appender.solr=org.apache.log4j.RollingFileAppender
-log4j.appender.solr.layout=org.apache.log4j.PatternLayout
-log4j.appender.solr.layout.ConversionPattern=%d{ISO8601} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
-log4j.appender.solr.file=${karaf.data}/log/solr.log
-log4j.appender.solr.append=true
-log4j.appender.solr.maxFileSize=20MB
-log4j.appender.solr.maxBackupIndex=10
-
-# Artemis File appender
-log4j.appender.artemis=org.apache.log4j.RollingFileAppender
-log4j.appender.artemis.layout=org.apache.log4j.PatternLayout
-log4j.appender.artemis.layout.ConversionPattern=%d{ISO8601} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
-log4j.appender.artemis.file=${karaf.data}/log/artemis.log
-log4j.appender.artemis.append=true
-log4j.appender.artemis.maxFileSize=20MB
-log4j.appender.artemis.maxBackupIndex=10
-
-# Sift appender
-log4j.appender.sift=org.apache.log4j.sift.MDCSiftingAppender
-log4j.appender.sift.key=bundle.name
-log4j.appender.sift.default=karaf
-log4j.appender.sift.appender=org.apache.log4j.FileAppender
-log4j.appender.sift.appender.layout=org.apache.log4j.PatternLayout
-log4j.appender.sift.appender.layout.ConversionPattern=%d{ABSOLUTE} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %m%n
-log4j.appender.sift.appender.file=${karaf.data}/log/$\\{bundle.name\\}.log
-log4j.appender.sift.appender.append=true
+log4j2.rootLogger.level = INFO
+log4j2.rootLogger.appenderRef.out.ref = out
+log4j2.rootLogger.appenderRef.osgi-all.ref = osgi-all
+
+# Loggers configuration
```

Comment:
This is how karaf formats this file: https://github.com/apache/karaf/blob/karaf-4.1.1/assemblies/features/base/src/main/resources/resources/etc/org.ops4j.pax.logging.cfg#L34.